### DISCUSSION TITLE
AZ-900 Episode 12: Core Database Services | Cosmos DB, SQL Database, SQL DB for MySQL and PostrgeSQL

### DISCUSSION THREAD

**Discussion by MarczakIO** [AzureCommunityPL/group#46] [Community Content]
Kolejny dzień kolejny odcinek. Tym razem opowiadam o podstawowych usługach bazodanowych w Azure do składowania danych częściowo ustrukturyzowanych oraz ustrukturyzowanych. Czyli pokrótce pare słów o Azure Cosmos DB, Azure SQL Database, Managed Instance, Database for MySQL, Database for PostgreSQL, etc.

EN: After exploring storage services we can move to a more structured way of storing data with a Core Database Services in Azure.
In Episode 12 of my Azure Fundamentals course we will learn about Azure Cosmos DB, SQL Database, SQL Datawarehouse, Database for MySQL, and PostgreSQL.

![](https://marczak.io/images/youtube/az-900-ep-12-740.jpg)

📺 Video: https://youtu.be/RqD4nMyBazU
🌐 Site: https://marczak.io/az-900/#ep12
💬 Practice Test: https://marczak.io/az-900/episode-12/practice-test/

**Comment by kmadof** [comment]
@MarczakIO literówka w tytule w `PostreSQL` :stuck_out_tongue_winking_eye:

**Reply by MarczakIO** [reply]
Ops... na YT zauwazylem i poprawilem ale zapomnialem ze tu copy-pasta poszla :) thnx!
